setup.php

```diff
@@ -40,7 +40,7 @@
    ------------------------------------------------------------------------
  */
 
-define ("PLUGIN_FUSIONINVENTORY_VERSION", "0.90+1.1");
+define ("PLUGIN_FUSIONINVENTORY_VERSION", "0.90+1.2");
 
```

Comment:
Can you first keep version "0.90+1.1" to see if tests pass? or you need modify unit test too